plugins/outputs/wavefront/sample.conf

```diff
@@ -50,3 +50,11 @@
   ## of metrics will block for a longer time, but this will be handled gracefully by the internal buffering in
   ## Telegraf.
   #immediate_flush = true
+
+  ## Optional TLS Config
+  #tls_ca = "/etc/telegraf/ca.pem"
+  #tls_cert = "/etc/telegraf/cert.pem"
+  #tls_key = "/etc/telegraf/key.pem"
```

Comment:
@srebhan has a PR (#13348) which will help standardize the optional TLS settings used across plugins. I think it would be good to include them all?

```
  ## Set to true/false to enforce TLS being enabled/disabled. If not set,
  ## enable TLS only if any of the other options are specified.
  # tls_enable =
  ## Trusted root certificates for server
  # tls_ca = "/path/to/cafile"
  ## Used for TLS client certificate authentication
  # tls_cert = "/path/to/certfile"
  ## Used for TLS client certificate authentication
  # tls_key = "/path/to/keyfile"
  ## Send the specified TLS server name via SNI
  # tls_server_name = "kubernetes.example.com"
  ## Use TLS but skip chain & host verification
  # insecure_skip_verify = false
```

Reply:
Thanks @powersj - it sounds like that PR will eventually generate the related `sample.conf` and `README` sections, so for now the thing to do is manually include all the relevant fields.